tests/journald_tests.cpp

```diff
@@ -292,6 +293,189 @@ TEST_F(JournaldLoggerTest, ROOT_LogToJournaldWithBigLabel)
 }
 
 
+// This checks a specific case where the arguments passed into the
+// ContainerLogger will differ if the Mesos agent restarts before
+// launching a nested container.
+TEST_F(JournaldLoggerTest, ROOT_CGROUPS_LaunchThenRecoverThenLaunchNested)
+{
+  mesos::internal::slave::Flags flags = CreateSlaveFlags();
+  flags.launcher = "linux";
+  flags.isolation = "cgroups/cpu,filesystem/linux,namespaces/pid";
```

Comment:
These are the minimal set of isolators/launchers we typically use for containers that support nesting.  There are a few more that could be enabled and we technically don't need the PID namespaces.  But it doesn't particularly hurt either way.